stackdriver/storage.go

```diff
@@ -72,7 +73,27 @@ func (s *Storage) Close() error {
 // ApplyConfig updates the state as the new config requires.
 func (s *Storage) ApplyConfig(conf *config.Config) error {
 	var projectId string
-	if value, ok := conf.GlobalConfig.ExternalLabels[ProjectIdLabel]; !ok {
+	var location string
+	var clusterName string
```

Comment:
In the most recent commit, the assignment to this variable was removed, leaving it blank throughout this function. The same is true for location as well. Was this an accidental deletion?